Content.Server/GameObjects/EntitySystems/ApcNetSystem.cs

```diff
@@ -34,5 +35,10 @@ public void RemoveApcNet(ApcNetNodeGroup apcNet)
         {
             _apcNets.Remove(apcNet);
         }
+
```

Comment:
Put in a doc comment here about why this is necessary.
```suggestion
    /// <summary>
    ///     <see cref="NodeGroupSystem"/> does not remake <see cref="ApcNet"/>s affected during restarting until a frame later, when their grid is invalid. So, we are clearing them on round restart.
    /// </summary>
```

Reply:
Doc comments aren't really meant for this, a regular comment should do fine.